cirq-web/README.rst

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
nit: remove new lines on the top

Reply:
We should also add the Cirq logo here. This is the description that gets published to PyPi.